airflow/ui/src/components/Select.tsx

```diff
@@ -0,0 +1,259 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/* eslint-disable no-underscore-dangle */
+import React from 'react';
+import Select, { components as selectComponents, Props as SelectProps } from 'react-select';
+import {
+  Flex,
+  Tag,
+  TagCloseButton,
+  TagLabel,
+  Divider,
+  CloseButton,
+  Center,
+  Box,
+  Portal,
+  StylesProvider,
+  useMultiStyleConfig,
+  useStyles,
+  useTheme,
+  useColorModeValue,
+  RecursiveCSSObject,
+  CSSWithMultiValues,
+} from '@chakra-ui/react';
+import { ChevronDownIcon } from '@chakra-ui/icons';
```

Comment:
We should just continue utilizing `react-icons` instead of adding a new dependency.
```suggestion
import { FiChevronDown } from 'react-icons/fi';
```